tools/integration_tests/operations/move_file_test.go

```diff
@@ -78,8 +88,6 @@ func TestMoveFileWithinSameDirectory(t *testing.T) {
 	}
 
 	checkIfFileMoveOperationSucceeded(filePath, destDirPath, t)
-
-	os.RemoveAll(dirPath)
```

Comment:
why are we removing this ? (here and at all other places)

Reply:
So now we are already cleaning before running any tests. so there is no need of cleaning after running tests(that's what I feel)

Before we were cleaning for some set of tests where we required like (list, rename, move)